docs/gallery/External_Resources.py

```diff
@@ -85,112 +157,58 @@
 # ------------------------------------------------------
-# This is another wrapper function, but allows the use of a pandas DataFrame
-# to add/reference keys, resources, and entities.
-
-new_data = {
-    'key_name': 'Homo sapiens',
-    'resources_idx': 0,
-    'entity_id': 'NCBI:txid9606',
-    'entity_uri': 'https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=9606'}
+# Sometimes you want to use an existing key when adding new ontology data into ``ExternalResources``.
+# The user must use the actual :py:class:`~hdmf.common.resources.Key` object and not the 'key_name'.
 
-df = pd.DataFrame(new_data, index=[0])
+er.add_ref(container=genotypes, field='genotype_name', key=key_object, resource_name='Ensembl',
+           resource_uri='https://uswest.ensembl.org/index.html', entity_id='ENSG00000198963',
+           entity_uri='https://uswest.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000198963')
 
-er.add_keys(df)
-
-###############################################################################
-# Using the add_resource method
-# ------------------------------------------------------
-# This method creates new resource objects and adds them to the ResourceTable.
-# The ResourceTable holds the name and uri for the resource.
-# For example, you could add NCBI Taxonomy with its uri.
-
-er = ExternalResources(name='new_example')
-
-
-er.add_resource(resource='NCBI_Taxonomy', uri='https://www.ncbi.nlm.nih.gov/taxonomy')
-
-# This will return a pandas dataframe of the ResourceTable
-er.resources.to_dataframe()
-
-###############################################################################
-# Using the add_entity method
-# ------------------------------------------------------
-# This method creates a new entity object and adds it to the EntityTable.
-# Keeping with the NCBI Taxonomy example, the entity would be a specfic item
-# or "search" in NCBI Taxonomy, whereas the resource is NCBI Taxonomy itself.
-
-er.add_entity(key='Homo sapiens', resources_idx=0,
-              entity_id='NCBI:txid9606',
-              entity_uri='https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=9606')
-
-er.add_entity(key='Mus musculus', resources_idx=0,
-              entity_id='NCBI:txid10090',
-              entity_uri='https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=10090')
-
-# This will return a pandas dataframe of the EntityTable
-er.entities.to_dataframe()
+# Let's use get_keys to visualize
+er.get_keys()
 
 ###############################################################################
-# Using the add_object method
+# Using get_object_resources
 # ------------------------------------------------------
-# This method creates a new object and adds it to the ObjectTable.
-
-er.add_object(container=data, field='Data')
+# This method will return information regarding keys, resources and entities for
+# an ``Object``. You can use either the name of the container or the container itself,
+# along with the field.
```

Comment:
How does one use the name of the container? Do you mean the object ID of the container?